src/Sulu/Bundle/AdminBundle/Resources/js/containers/MultiSelect/MultiSelect.js

```diff
@@ -78,11 +77,11 @@ export default class MultiSelect<T: string | number> extends React.Component<Pro
                 onChange={this.handleChange}
                 values={values}
             >
-                {this.resourceListStore.data.map((object, index) => (
+                {this.resourceListStore.data.map((object, index) => ((
                     <MultiSelectComponent.Option key={index} value={object[idProperty]}>
                         {object[displayProperty]}
                     </MultiSelectComponent.Option>
-                ))}
+                ): Element<typeof MultiSelectComponent.Option>))}
```

Comment:
Why is that necessary? Is that a way to circumvent a flow bug?

Reply:
yes, flow infers Element<any> as return type of this arrow function and throws an error.